docs/DeveloperGuide.adoc

```diff
@@ -114,99 +186,489 @@ NOTE: The lifeline for `DeleteCommandParser` should end at the destroy marker (X
  +
 image:BetterModelClassDiagram.png[]
 
 [[Design-Storage]]
 === Storage component
 
+The following class diagram below displays the structure of the Storage Component.
+
 .Structure of the Storage Component
 image::StorageClassDiagram.png[]
 
-*API* : link:{repoURL}/src/main/java/seedu/address/storage/Storage.java[`Storage.java`]
+*API* : link:{repoURL}/src/main/java/seedu/moneygowhere/storage/Storage.java[`Storage.java`]
 
 The `Storage` component,
 
 * can save `UserPref` objects in json format and read it back.
-* can save the Address Book data in json format and read it back.
+* can save the Spending Book data in json format and read it back.
 
 [[Design-Commons]]
 === Common classes
 
-Classes used by multiple components are in the `seedu.addressbook.commons` package.
+Classes used by multiple components are in the `seedu.moneygowhere.commons` package.
 
 == Implementation
 
 This section describes some noteworthy details on how certain features are implemented.
 
+=== Logging
+
+We are using `java.util.logging` package for logging. The `LogsCenter` class is used to manage the logging levels and logging destinations.
+
+* The logging level can be controlled using the `logLevel` setting in the configuration file (See <<Implementation-Configuration>>)
+* The `Logger` for a class can be obtained using `LogsCenter.getLogger(Class)` which will log messages according to the specified logging level
+* Currently log messages are output through: `Console` and to a `.log` file.
+
+*Logging Levels*
+
+* `SEVERE` : Critical problem detected which may possibly cause the termination of the application
+* `WARNING` : Can continue, but with caution
+* `INFO` : Information showing the noteworthy actions by the App
+* `FINE` : Details that is not usually noteworthy but may be useful in debugging e.g. print the actual list instead of just its size
+
+[[Implementation-Configuration]]
+=== Configuration
+
+Certain properties of the application can be controlled (e.g user prefs file location, logging level) through the configuration file (default: `config.json`).
+
+
+[[Implementation-Find]]
+=== Find feature
+
+The find feature allows the user to search for a spending based on specified field. If search results are too broad, fields may be combined to increase specificity.
+
+For example, `find n/Apple c/2.50-3.00` will find an `Apple` of cost range $`2.50` to $`3.00`.
+
+==== Implementation
+
+Find is supported by having a `Predicate` implemented for every field in Spending. Predicates are added based on valid input entered by the user.
+
+The `FindCommandParser` class stores these predicates, which are combined using Java 8 streams with an `AND` operation to form a more specific search query.
+
+The sequence diagram below demonstrates how the `find` command is executed:
+
+.Sequence diagram for an example `find` command
+image::FindSequenceDiagram.png[Find Sequence Diagram,width=75%]
```

Comment:
![image](https://user-images.githubusercontent.com/54146425/67548143-a67b4500-f733-11e9-86aa-ed046bb982f6.png)
May want to include a similar disclaimer:
![image](https://user-images.githubusercontent.com/54146425/67548177-c27ee680-f733-11e9-9a84-161ce90d22cb.png)
for FindCommand